.travis.yml

```diff
@@ -1,17 +1,15 @@
 language: objective-c
-osx_image: xcode7.2
+osx_image: xcode7.3
 before_install:
 - scripts/import_keychain.sh
 - security find-identity -v -p codesigning
-- brew update
-- brew install carthage
```

Comment:
In Xcode image, Carthage is already installed.

https://blog.travis-ci.com/2016-03-23-xcode-image-updates